site/quorum-queue.md

```diff
@@ -0,0 +1,216 @@
+<!--
+Copyright (c) 2007-2018 Pivotal Software, Inc.
+
+All rights reserved. This program and the accompanying materials
+are made available under the terms of the under the Apache License,
+Version 2.0 (the "License”); you may not use this file except in compliance
+with the License. You may obtain a copy of the License at
+
+http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+
+# Quorum Queue NOSYNTAX
+
+## Overview
+
+The quorum queue is a new (since 3.8) queue type for RabbitMQ implementing a persistent,
+replicated fifo queue using [the Raft consensus algorithm](https://raft.github.io/).
+
+The quorum queue type is an alternative to persistent HA queues for use cases
```

Comment:
Should we use "HA queues" or "mirrored queues"? I don't know the official name, but I'm more used to "mirrored queues" personally.

Reply:
Good question. If you feel "mirrored queues" is the more common term I can use that instead. Or perahps I could write "Highty Available (Mirrored) queues" just like the doc page: https://www.rabbitmq.com/ha.html :)

Reply:
Yes, I think "highly available (mirrored) queues" with a link to the section for the first reference would work. Then simply "HA queues" is good. Maybe stick with the link to the section, at least for the latest references.